src/clientdlg.cpp

```diff
@@ -258,7 +258,13 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     lblGlobalInfoLabel->hide();
 
     // prepare update check info label (invisible by default)
-    lblUpdateCheck->setText ( "<font color=\"red\"><b>" + QString ( APP_NAME ) + " " + tr ( "software upgrade available" ) + "</b></font>" );
+    lblUpdateCheck->setOpenExternalLinks ( true ); // enables opening a web browser if one clicks on a html link
+    lblUpdateCheck->setText (
+        "<font color=\"#c94a55\"><b>" +
+        tr ( APP_UPGRADE_AVAILABLE_MSG_TEXT )
```

Comment:
This won't work (it won't show up as a translatable string). It should be just:

`APP_UPGRADE_AVAILABLE_MSG_TEXT`

No 'tr' nor brackets. See further suggested changes to make it translatable.